Jenkinsfile_CNP

```diff
@@ -6,7 +6,7 @@ def type = "java"
 def product = "darts"
 def component = "api"
 
-def branchesToSync = ['demo']
+def branchesToSync = ['sbox']
```

Comment:
We still want demo as well

Reply:
Plus Dave has already done on gateay service like this https://github.com/hmcts/darts-gateway/blame/ccaf77a060baba4ae485e8d89ed957b4aad36619/Jenkinsfile_CNP#L9 
Can we keep the naming consistent so we don't create confusion?

Reply:
sbox might well be right?! 🤷‍♂️

Reply:
I've just being going off the files in the flux repo for it, happy to change it though ?